dev/devicelab/manifest.yaml

```diff
@@ -150,6 +150,12 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["has-android-device"]
 
+  e2e:
+    description: >
+      Runs end-to-end Flutter tests on Android.
+    stage: devicelab
+    required_agent_capabilities: ["has-android-device"]
```

Comment:
Are we confident that the tests will be green in the lab immediately? If not, consider marking as `flaky: true`, iterate until green, then unmarking it.